scripts/tests-integration-stage.sh

```diff
@@ -43,7 +43,7 @@ SCRIPT_PATH=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
 cd ${SCRIPT_PATH}/..
 
 # CHECK ALV2 DISCLAIMER
-if [ $(find src -name "*.java" -exec grep -L Licensed {} \; | wc -l) -gt 0 ]; then
+if [ $(find */src -name "*.java" -exec grep -L Licensed {} \; | wc -l) -gt 0 ]; then
```

Comment:
Already fixed in https://github.com/gisaia/ARLAS-server/pull/293/files
... and not relevant for Tiles service 😉

Reply:
It was not fixed in https://github.com/gisaia/ARLAS-server/pull/293/files (half of it is missing) and is needed for checking the disclaimer of the tile service code.